client/index.html

```diff
@@ -4,9 +4,27 @@
 		<meta charset="UTF-8" />
 		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
 		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
-		<title>Document</title>
+		<title>Home</title>
 		<script defer src="./assets/bundle.js"></script>
+		<link rel="preconnect" href="https://fonts.googleapis.com">
+		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+		<link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
 		<link rel="stylesheet" href="assets/css/style.css" />
 	</head>
-	<body></body>
+	<body>
+		<nav>
+			<h4 id="h4" class="navbar-grid-item">JOURNEL</h4>
+			<div class="navbar-grid-item" id='navbar-grid'>
```

Comment:
Having an id an a class here is potentially redundant? 
Consider the level of specificity required

Reply:
See earlier response to the h4 id, por favor :)